ONSdigital/dp-api-router
# Add routing for articles API

### What

Route `/articles` requests to the articles API, sitting behind a feature flag

### How to review

Check code and tests.

Optionally test it:
- Run zebedee in reader mode and articles API
- Run API router with flag on: `export ENABLE_ARTICLES_API=true && make debug`
- Send a GET request with a valid article url. For example: `localhost:23200/v1/articles/legacy?url=/economy/environmentalaccounts/bulletins/ukenvironmentalaccounts/2015-07-09` - It should return the article data
- Run API router with flag off: `export ENABLE_ARTICLES_API=false && make debug`
- Send a GET request with a valid article url. For example: `localhost:23200/v1/articles/legacy?url=/economy/environmentalaccounts/bulletins/ukenvironmentalaccounts/2015-07-09` - It should return 404


### Who can review

Anyone
